exporter/splunkhecexporter/client.go

```diff
@@ -95,7 +95,7 @@ func (c *client) pushMetricsData(
 	if md.ResourceMetrics().Len() != 0 {
 		accessToken, found := md.ResourceMetrics().At(0).Resource().Attributes().Get(splunk.HecTokenLabel)
 		if found {
-			localHeaders["Authorization"] = splunk.HECTokenHeader + " " + accessToken.Str()
+			localHeaders["Authorization"] = splunk.HecTokenHeader + " " + accessToken.Str()
```

Comment:
Please revert that change and offer it in a separate PR if you want, so we can have a smaller diff to review.